### PR TITLE
Replace REQUIRE by Project.toml

### PR DESCRIPTION
**Pull request by juliohm**
I appreciate if you can review this PR and merge as soon as possible. I believe VisualRegressionTests.jl is holding some of my package tests somehow because of the old REQUIRE system.

### CODE REVIEW

**Comment by juliohm** [comment]
Please add JuliaRegistrator and JuliaTagBot to this repo if it is not already added :+1:

**Comment by daschw** [comment]
I think it would make sense to add Manifest.toml to gitignore, no?
JuliaRegistrator and JuliaTagBot are already added to JuliaPlots.

**Comment by juliohm** [comment]
I am not using Manifest.toml at this point, but I can add a commit if you like.

**Comment by daschw** [comment]
That would be great

**Comment by juliohm** [comment]
Done

**Comment by asinghvi17** [comment]
@juliohm we already have TagBot and Registrator configured organization-wide.

**Comment by daschw** [comment]
Thanks @juliohm !

**Comment by daschw** [comment]
@JuliaRegistrator register()

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+Manifest.toml
```
Comment (daschw) [inline .gitignore]: @JuliaRegistrator register()

**Pull request merged**